packages/desktop-client/src/components/accounts/TransactionsTable.test.js

```diff
@@ -8,22 +8,28 @@ import {
   generateTransaction,
   generateAccount,
   generateCategoryGroups,
-  TestProvider,
 } from 'loot-core/src/mocks';
+import { TestProvider } from 'loot-core/src/mocks/redux';
 import { initServer } from 'loot-core/src/platform/client/fetch';
 import {
   addSplitTransaction,
   realizeTempTransactions,
   splitTransaction,
   updateTransaction,
-} from 'loot-core/src/shared';
+} from 'loot-core/src/shared/transactions';
 import { integerToCurrency } from 'loot-core/src/shared/util';
-import { SelectedProviderWithItems } from 'loot-design/src/components';
+import { SelectedProviderWithItems } from 'loot-design/src/components/useSelected';
+
+import installPolyfills from '../../polyfills';
 
 import { SplitsExpandedProvider, TransactionTable } from './TransactionsTable';
 
 const uuid = require('loot-core/src/platform/uuid');
 
+jest.mock('loot-core/src/platform/client/fetch');
+
+installPolyfills();
```

Comment:
Do we need to `installPolyfills()` elsewhere during test runs? Could it be done once in test setup for all tests?

Reply:
Makes sense: https://github.com/actualbudget/actual/pull/697/commits/0c613215b09cbd399c4c5f9a6a523ec6654d583b

Currently we have only 1x test in the `desktop-client` package, but we might have more in the future. So it makes sense making this right.